supplementary/alica_ros1/alica_tests/src/test/test_assignment.cpp

```diff
@@ -95,4 +95,5 @@ TEST(Assignment, RobotsInserted)
         ++i;
     }
     ASSERT_EQ(i, 2);
+    ac->terminate();
```

Comment:
Destruction doesn't terminate?

Reply:
no it will be called by dtor if it was initialized but not terminated